projects/pt1/python/torch_mlir_e2e_test/configs/torchdynamo.py

```diff
@@ -54,6 +54,32 @@ def _returns_empty_tuple(fx_graph: torch.fx.GraphModule) -> bool:
     return True
 
 
+def transform(gm: torch.fx.GraphModule):
```

Comment:
Please give this a descriptive name and comment. Something like scalarize_tensor_ops_on_scalars.

Reply:
Done, for some reason I can't check e2e tests locally. Maybe "AtenInstanceNormModule_basic" should be added to fixed, but I can't check, have this error:
```
$ python -m e2e_testing.main -f "AtenInstanceNormModule_basic"
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "torch-mlir/projects/pt1/e2e_testing/main.py", line 16, in <module>
    from torch_mlir_e2e_test.configs import (
  File "torch-mlir/build/tools/torch-mlir/python_packages/torch_mlir/torch_mlir_e2e_test/configs/__init__.py", line 6, in <module>
    from .lazy_tensor_core import LazyTensorCoreTestConfig
  File "torch-mlir/build/tools/torch-mlir/python_packages/torch_mlir/torch_mlir_e2e_test/configs/lazy_tensor_core.py", line 6, in <module>
    import torch_mlir._mlir_libs._REFERENCE_LAZY_BACKEND as lazy_backend
  File "torch-mlir/build/tools/torch-mlir/python_packages/torch_mlir/torch_mlir/_mlir_libs/__init__.py", line 180, in <module>
    _site_initialize()
  File "torch-mlir/build/tools/torch-mlir/python_packages/torch_mlir/torch_mlir/_mlir_libs/__init__.py", line 78, in _site_initialize
    from ._mlir import ir
ImportError: cannot import name 'ir' from 'torch_mlir._mlir_libs._mlir' (unknown location)
```

Reply:
sorry, resolved my issue with environment